graphrag_sdk/models/litellm.py

```
@@ -53,9 +54,14 @@ def __init__(
         if not self.check_valid_key(model_name):
             raise ValueError(f"Invalid keys for model {model_name}.")
         
-
+        if self.model_name == "gpt-4.1":
+            # Set default temperature to 0 for gpt-4.1
+            if generation_config is None:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)